examples/README.md

```diff
@@ -29,3 +29,27 @@ This ASP.NET web application is a .NET Framework MVC 5 app. It leverages the [co
 
 ## [WebFormApp (.NET Framework)](./DotNetFramework/WebFormApp)
 This ASP.NET web application is a .NET Framework Web Forms app. It demonstrates how to leverage the App Configuration [.NET Standard provider library](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration/) to achieve dynamic configuration and control feature launches with feature flags. The same technique applies to .NET Framework MVC apps.
+
+## [Spring Samples](https://github.com/Azure-Samples/azure-spring-boot-samples/)
+
+Contains multiple Azure App Configuration Spring boot sample projects.
+
+### azure-appconfiguration-sample
+
+This example shows how to use the basic features of Azure App Configuration
+
+### azure-appconifugration-refresh-sample
+
+This example shows how to dynamically update configurations in a Spring Web application.
```

Comment:
configuration instead of configurations. It is gramatically correct either way but configuration makes better sense as we have only one store.

Reply:
configuration -> application settings